ee/secureenclavesigner/secureenclavesigner_test.go

```diff
@@ -0,0 +1,166 @@
+//go:build darwin
+// +build darwin
+
+package secureenclavesigner
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/kolide/krypto/pkg/echelper"
+	"github.com/kolide/launcher/ee/agent/storage/inmemory"
+	"github.com/kolide/launcher/pkg/log/multislogger"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testWrappedEnvVarKey = "SECURE_ENCLAVE_TEST_WRAPPED"
+	macOsAppResourceDir  = "./test_app_resources"
+)
+
+func WithBinaryPath(p string) opt {
+	return func(ses *secureEnclaveSigner) {
+		ses.pathToLauncherBinary = p
+	}
+}
+
+// #nosec G306 -- Need readable files
+func TestSecureEnclaveSigner(t *testing.T) {
+	t.Parallel()
+
+	if os.Getenv("CI") != "" {
+		t.Skipf("\nskipping because %s env var was not empty, this is being run in a CI environment without access to secure enclave", testWrappedEnvVarKey)
+	}
+
+	// put the root dir somewhere else if you want to persist the signed macos app bundle
+	// should build this into make at some point
+	rootDir := "/tmp/secure_enclave_test"
+
+	// rootDir := t.TempDir()
```

Comment:
This should probably be using `t.TempDir()` as the commented code implies

Reply:
addressed in https://github.com/kolide/launcher/pull/1647